README.md

```diff
@@ -131,6 +142,15 @@ scrape_configs:
         regex: .*
         target_label: __address__
         replacement: 127.0.0.1:9115  # Blackbox exporter.
+
+  - job_name: 'blackbox-post'
+    metrics_path: /probe
+    params:
+      body: ['{}'] # Content, type must match the Content-Type header configured in the module
```

Comment:
This is how we approach everything for the blackbox exporter. Each config is meant to be a test of a particular thing such as a service, which you then run against multiple targets.